src/redisearch_rs/buffer/src/lib.rs

```diff
@@ -121,3 +121,13 @@ impl Buffer {
         self.0.offset += n;
     }
 }
+
+impl std::fmt::Debug for Buffer {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("Buffer")
+            .field("len", &self.len())
+            .field("capacity", &self.capacity())
+            .field("data", &self.as_slice())
```

Comment:
Do we want to print all the data in a debug?

Reply:
I'm worried about a 1Kb slice being printed to the terminal if this is accidentally called in prod code. Even worse if it contains sensitive data.